atomkraft/cli/__init__.py

```diff
@@ -2,9 +2,11 @@
 
 import pytest
 import typer
+import os
 from copier import run_auto
 
 from .. import chain
+from .. import Reactor
```

Comment:
Python convention for naming modules is [lowercase names with underscores](https://peps.python.org/pep-0008/#package-and-module-names).

Please rename `atomkraft/Reactor.py` to `atomkraft/reactor.py` and update the imports accordingly.